graph.go

```diff
@@ -88,6 +88,10 @@ type Graph[K comparable, T any] interface {
 	// doesn't exist.
 	Vertex(hash K) (T, error)
 
+	// Vertices returns a slice of all vertices in the graph. These vertices are of type
+	// Vertice[K] and hence will contain the vertex hashes, not the vertex values.
```

Comment:
should this read `K` rather than `Vertice[K]`?

Reply:
> should this read `K` rather than `Vertice[K]`?

@alexnguyenatwork Yes. There is no `Vertex` type (let alone `Vertice`).